index.html

```diff
@@ -203,6 +203,51 @@ <h3>Skills</h3>
           </div>
         </section>
       </section>
+      <section id="form-section">
+        <h2 class="form-heading">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+        <form action="https://formspree.io/f/mdovqere" method="post">
+          <label for="name">
+            <input id="name" name="name" type="text" maxlength="30" placeholder="Full name"required>
+          </label>
+          <label for="email">
+            <input id="email" name="email" type="email" placeholder="Email address" required>
+          </label>
+          <label for="message">
+            <textarea name="message" maxlength="500" required>Write me something...</textarea>
```

Comment:
- [ ] Good job for using `textarea` ✌ Please make sure to add the correct associated `placeholder` text instead of directly inserting it here. You can do it the same you you did for other input fields ⛏🤓 This is also mentioned in the project requirements 👇

  > _All fields have placeholder text._